styles/widgets/win8/dataGrid.win8.less

```diff
@@ -594,16 +594,16 @@
         .text-shadow(none);
         color: @WIN8_DATAGRID_SELECTION_COLOR;
 
-        & > td {
+        & > td, & > tr > td {
             background-color: @WIN8_DATAGRID_SELECTION_BACKGROUND;
 
             &.dx-datagrid-group-space {
                 border-right-color: @WIN8_DATAGRID_SELECTION_BACKGROUND;
             }
-        }
 
-        & > td.dx-focused {
-            background-color: transparent;
+            &.dx-focused {
```

Comment:
td selector is removed. Is this change correct?

Reply:
Yes, it is correct, because now  it in the scope of previous selector (it is not necessary to duplicate it)